Command update - spo tenant settings set (#5352)
pnp/cli-microsoft365: docs/docs/cmd/spo/tenant/tenant-settings-set.mdx

```md definition-list
`--MinCompatibilityLevel [MinCompatibilityLevel]`
: Specifies the lower bound on the compatibility level for new sites.

`--MaxCompatibilityLevel [MaxCompatibilityLevel]`
: Specifies the upper bound on the compatibility level for new sites.

`--ExternalServicesEnabled [ExternalServicesEnabled]`
: Enables external services for a tenant. External services are defined as services that are not in the Microsoft 365 data centers. Allowed values `true`, `false`.

`--NoAccessRedirectUrl [NoAccessRedirectUrl]`
: Specifies the URL of the redirected site for those site collections which have the locked state "NoAccess".

`--ArchiveRedirectUrl [ArchiveRedirectUrl]`
: Custom URL where the users will be taken if they select 'Request to reactivate' when they encounter archived content. The button won't be visible if there is no redirect URL set.

`--ConditionalAccessPolicyErrorHelpLink [ConditionalAccessPolicyErrorHelpLink]`
: Sets the link to organization help page in case of access denied due to conditional access policy.

`--CustomizedExternalSharingServiceUrl [CustomizedExternalSharingServiceUrl]`

> : Specifies a URL that will be appended to the error message that is surfaced when a user is blocked from sharing externally by policy. This URL can be used to direct users to internal portals to request help or to inform them about your organization's policies. An example value is "https://www.contoso.com/sharingpolicies".

Author: Pushed commit with required changes.

`--CoreSharingCapability [CoreSharingCapability]`
: Configure the sharing level for SharePoint sites (not including OneDrive sites). Allowed values: `ExternalUserAndGuestSharing` (default) - External user sharing (share by email) and guest link sharing are both enabled, `Disabled` - External user sharing (share by email) and guest link sharing are both disabled, `ExternalUserSharingOnly` - External user sharing (share by email) is enabled but guest link sharing is disabled, `ExistingExternalUserSharingOnly` - Only guests already in your organization's directory.

`--ODBSharingCapability [ODBSharingCapability]`
: Configure the sharing level for for OneDrive sites. Allowed values: `ExternalUserAndGuestSharing` (default) - External user sharing (share by email) and guest link sharing are both enabled, `Disabled` - External user sharing (share by email) and guest link sharing are both disabled, `ExternalUserSharingOnly` - External user sharing (share by email) is enabled but guest link sharing is disabled, `ExistingExternalUserSharingOnly` - Only guests already in your organization's directory.

`--OneDriveLoopSharingCapability [OneDriveLoopSharingCapability]`

> : Configure the sharing level for the loop components hosted on OneDrive, for example: whiteboards in Teams meetings. Allowed values: `ExternalUserAndGuestSharing` (default) - External user sharing (share by email) and guest link sharing are both enabled, `Disabled` - External user sharing (share by email) and guest link sharing are both disabled, `ExternalUserSharingOnly` - External user sharing (share by email) is enabled but guest link sharing is disabled, `ExistingExternalUserSharingOnly` - Only guests already in your organization's directory.

Author: Pushed commit with required changes.

`--CoreDefaultShareLinkRole [CoreDefaultShareLinkRole]`
: Sets the default share link role on core partition. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

`--CoreLoopDefaultSharingLinkRole [CoreLoopDefaultSharingLinkRole]`
: Sets the default share link role for fluid on SharePoint sites. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

`--ContainerDefaultShareLinkRole [ContainerDefaultShareLinkRole]`
: Sets the default share link for SharePoint Embedded. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

`--ContainerLoopDefaultShareLinkRole [ContainerLoopDefaultShareLinkRole]`

> : Sets the default share link for Loop components. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

Author: Pushed the commit with required changes.

`--OneDriveDefaultShareLinkRole [OneDriveDefaultShareLinkRole]`
: Sets default share link role on OneDrive partition. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

`--OneDriveLoopDefaultSharingLinkRole [OneDriveLoopDefaultSharingLinkRole]`
: Sets default share link role for fluid on OneDrive sites. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`.

`--CoreDefaultShareLinkScope [CoreDefaultShareLinkScope]`
: Sets default share link scope on core partition. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`.

`--CoreLoopDefaultSharingLinkScope [CoreLoopDefaultSharingLinkScope]`
: Sets default share link scope for fluid on SharePoint sites. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`.

`--ContainerDefaultShareLinkScope [ContainerDefaultShareLinkScope]`

> : Sets default share link scope for SharePoint Embedded. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`.

Author: Pushed commit with required changes.

`--ContainerLoopDefaultShareLinkScope [ContainerLoopDefaultShareLinkScope]`

> : Sets the default share link scope for Loop components. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`.

Author: Pushed the commit with required changes.

: Specifies the number of days before a Request files link expires for all OneDrive sites. The value can be from 0 to 730 days. To remove the expiration requirement, set the value to zero (0).

`--ExternalUserExpirationRequired [ExternalUserExpirationRequired]`
: Specifies whether to enable the external user expiration policy, where external users will be expired and removed from the site collection in a given number of days. Allowed values `true`, `false`.

`--ExternalUserExpireInDays [ExternalUserExpireInDays]`
: Specifies the number of days before an external user will expire and be removed from the site collection if the policy is enabled. Value can be from 30 to 730 days.

`--ReduceTempTokenLifetimeEnabled [ReduceTempTokenLifetimeEnabled]`
: Enables reduced session timeout for temporary URLs used by apps for document download scenarios. Reduction occurs when an app redeeming an IP address does not match the original requesting IP. Allowed values `true`, `false`.

`--ReduceTempTokenLifetimeValue [ReduceTempTokenLifetimeValue]`
: Sets the session timeout value for temporary URLs. The value can be set between 5 and 15 minutes and the default value is 15 minutes.

> .

Reviewer: This period doesn't belong here.

Author: I found this period from this command: https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spotenant?view=sharepoint-ps#-reducetemptokenlifetimevalue. Can you check and confirm once? Should I remove it from description or keep it?

Reviewer: I'm talking about the period character on line 243 😊

Author: Thanks, pushed new commit with required changes!

`--MediaTranscriptionAutomaticFeatures [MediaTranscriptionAutomaticFeatures]`
: When enabled, videos can have transcripts generated automatically on upload. Allowed values `Enabled`, `Disabled`.

`--ImageTaggingOption [ImageTaggingOption]`
: Specifies the image tagging option for the tenant. Allowed values `Basic`, `Enhanced`, `Disabled.

`--MarkNewFilesSensitiveByDefault [MarkNewFilesSensitiveByDefault]`
: If external sharing is turned on, sensitive content could be shared and accessed by guests before the Office DLP rule finishes processing, you can address this issue by configuring this option. Allowed values `AllowExternalSharing`, `BlockExternalSharing`.

`--OCRAdminSiteListFileName [OCRAdminSiteListFileName]`
: Sets the name of the file that contains the list of OCR-enabled sites for full-text search by the tenant admin.

`--OCRComplianceSiteListFileName [OCRComplianceSiteListFileName]`
: Sets the name of the file that contains the list of OCR-enabled sites for full-text search, managed by the compliance administrator.

`--OCRModeForAdminSites [OCRModeForAdminSites]`

> : Specifies whether the tenant admin has enabled OCR on SharePoint sites for full-text search through Syntex. Allowed values `Disabled`, `ExclusionList`, `InclusionList`.

Reviewer: ```suggestion
: Specifies whether the tenant admin has enabled OCR on SharePoint sites for full-text search. Allowed values `Disabled`, `ExclusionList`, `InclusionList`.
```

Author: Pushed commit with required changes.

`--OCRModeForComplianceODBs [OCRModeForComplianceODBs]`
: Specifies whether the compliance admin has enabled OCR for full text search through Syntex for ODBs. Allowed values `Disabled`, `ExclusionList`, `InclusionList`.

`--OCRModeForComplianceSites [OCRModeForComplianceSites]`
: Specifies whether the compliance admin has enabled OCR for full text search through Syntex for SharePoint sites. Allowed values `Disabled`, `ExclusionList`, `InclusionList`.

`--OneDriveDefaultLinkToExistingAccess [OneDriveDefaultLinkToExistingAccess]`
: Sets default share link to existing access on OneDrive partition. Allowed values `true`, `false`.

`--ContainerDefaultLinkToExistingAccess [ContainerDefaultLinkToExistingAccess]`

> : Sets default share link to existing access for SharePoint Embedded. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Sets the default share link for existing access for SharePoint Embedded. Allowed values `true`, `false`.
```

Author: Pushed commit with required changes.

`--ShowNGSCDialogForSyncOnODB [ShowNGSCDialogForSyncOnODB]`
: Shows NGSC dialog for sync on OneDrive for Business. Allowed values `true`, `false`.

`--SpecialCharactersStateInFileFolderNames [SpecialCharactersStateInFileFolderNames]`
: Sets the special characters state in file and folder names in SharePoint and OneDrive for Business. Allowed values `NoPreference`, `Allowed`, `Disallowed`.

`--SyncPrivacyProfileProperties [SyncPrivacyProfileProperties]`
: Syncs privacy profile properties. Allowed values `true`, `false`.

`--ExcludedFileExtensionsForSyncClient [ExcludedFileExtensionsForSyncClient]`
: Excluded file extensions for sync client. Array of strings split by comma.

`--AllowedDomainListForSyncClient [AllowedDomainListForSyncClient]`
: Sets allowed domain list for sync client. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.

`--DisabledWebPartIds [DisabledWebPartIds]`
: Sets disabled web part Ids. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.

`--DisabledModernListTemplateIds [DisabledModernListTemplateIds]`

> : Modern list template IDs to hide when creating a new list. An array of GUIDs split by a comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.

Reviewer: ```suggestion
: Modern list template IDs to hide when creating a new list. Specify a list of GUIDs split by a comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`. Specify an empty string to clear this value.
```

Author: Pushed commit with required changes.

`--CommentsOnListItemsDisabled [CommentsOnListItemsDisabled]`
: Enables or disables comments on lists. Allowed values `true`, `false`.

`--EnableAzureADB2BIntegration [EnableAzureADB2BIntegration]`
: Enables the preview for OneDrive and SharePoint integration with Azure AD B2B. Allowed values `true`, `false`. Azure AD one-time passcode needs to be enabled for this integration to work. For more information see [http://aka.ms/spo-b2b-integration](https://aka.ms/spo-b2b-integration).

`--EnableAutoNewsDigest [EnableAutoNewsDigest]`
: Enables or disables SharePoint auto news digest. This is an automated email to users about the latest news posts that are relevant to them, but that they may not have viewed yet. Allowed values `true`, `false`.

`--AllowCommentsTextOnEmailEnabled [AllowCommentsTextOnEmailEnabled]`
: Allows showing surrounding document context in email notifications when a user is mentioned. Allowed values `true`, `false`.

`--CommentsOnFilesDisabled [CommentsOnFilesDisabled]`
: Disables or enables commenting functionality on files. Allowed values `true`, `false`.

`--DisableAddToOneDrive [DisableAddToOneDrive]`

> : Disables or enables `Add shortcut to OneDrive` feature in SharePoint document libraries.  Allowed values `true`, `false`.

Reviewer: Can we also place this next to `HideSyncButtonOnODB` since these options are somewhat related?

```suggestion
: Disables or enables 'Add shortcut to OneDrive' feature in SharePoint document libraries.  Allowed values `true`, `false`.
```

Author: Pushed commit with required changes.

`--DisablePersonalListCreation [DisablePersonalListCreation]`
: Allows creating personal lists within the OneDrive for Business site of the user. Allowed values `true`, `false`.

`--ViewInFileExplorerEnabled [ViewInFileExplorerEnabled]`
: Enables or disables the ability to use View in Explorer in Microsoft Edge. Allowed values `true`, `false`.

`--AllowGuestUserShareToUsersNotInSiteCollection [AllowGuestUserShareToUsersNotInSiteCollection]`
: Allows guest users to share to users not in the site collection. Allowed values `true`, `false`.

`--BlockSendLabelMismatchEmail [BlockSendLabelMismatchEmail]`
: Allows blocking of the automated e-mail being sent when somebody uploads a document to a site that's protected with a sensitivity label and their document has a higher priority sensitivity label than the sensitivity label applied to the site. Allowed values `true`, `false`.

`--CoreDefaultLinkToExistingAccess [CoreDefaultLinkToExistingAccess]`
: Sets default share link to existing access on core partition. Allowed values `true`, `false`.

`--CoreRequestFilesLinkEnabled [CoreRequestFilesLinkEnabled]`

> : Enables or disables the Request files link on the core partition for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Enables or disables the 'Request files' link on the core partition for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
```

`--DisableDocumentLibraryDefaultLabeling [DisableDocumentLibraryDefaultLabeling]`
: Allows tenant admins to disable the capability of configuring a default sensitivity label for a document library. Allowed values `true`, `false`.

`--DisableVivaConnectionsAnalytics [DisableVivaConnectionsAnalytics]`
: Specifies whether Viva Connections Analytics is disabled or enabled on the tenant. Allowed values `true`, `false`.

`--DisplayNamesOfFileViewersInSpo [DisplayNamesOfFileViewersInSpo]`
: Allows configuring whether display name of people who view the file are visible in the property pane of the site in SharePoint site collection. Allowed values `true`, `false`.

`--EnableAIPIntegration [EnableAIPIntegration]`
:  Specifies whether Azure Information Protection (AIP) should be enabled on the tenant. Allowed values `true`, `false`.

`--EnableRestrictedAccessControl [EnableRestrictedAccessControl]`
: Enables restricted access control in SharePoint Online. You need to wait approximately 1 hour before managing restricted access control for a site collection. Allowed values `true`, `false`.

> `--HideSyncButtonOnDocLib [HideSyncButtonOnDocLib]`

Reviewer: Can we place this option next to `HideSyncButtonOnODB` so they are grouped together?

Author: Pushed commit with required changes.

> : Enables or disables the "Sync" button on all document libraries in SharePoint sites and OneDrive for Business sites. Allowed values `true`, `false`.

Reviewer: My bad, this is only for SharePoint.

```suggestion
: Enables or disables the "Sync" button on all document libraries in SharePoint sites. Allowed values `true`, `false`.
```

Author: Pushed commit with required changes.

`--NoAccessRedirectUrl [NoAccessRedirectUrl]`
: Specifies the URL of the redirected site for those site collections which have the locked state "NoAccess"

`--ArchiveRedirectUrl [ArchiveRedirectUrl]`
: Specifies the URL of the redirected site for those site collections which has been archived using Microsoft Syntex Archiving

`--ConditionalAccessPolicyErrorHelpLink [ConditionalAccessPolicyErrorHelpLink]`
: Sets the link to organization help page in case of access denied due to conditional access policy

`--CustomizedExternalSharingServiceUrl [CustomizedExternalSharingServiceUrl]`

> : Specifies a URL for "Help Link" in email with the subject "Incompatible sensitivity label detected" sent when a sensitivity label mismatch occurs between the label on the document uploaded and the label on the site

Author: Pushed commit with required changes.

`--LabelMismatchEmailHelpLink [LabelMismatchEmailHelpLink]`

> : Specifies a URL that will be appended to the error message that is surfaced when a user is blocked from sharing externally by policy

Author: Pushed commit with required changes.

`--SharingCapability [SharingCapability]`

> : Determines what level of sharing is available for the site. The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

Author: Added missing meaning for `ExistingExternalUserSharingOnly` value in the description.

`--CoreSharingCapability [CoreSharingCapability]`

> : Determines what level of sharing is available for SharePoint sites (not including OneDrive sites). The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

Author: Pushed commit with required changes.

`--ODBSharingCapability [ODBSharingCapability]`
: Determines what level of sharing is available for OneDrive sites. The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

`--OneDriveLoopSharingCapability [OneDriveLoopSharingCapability]`

> : Determines what level of sharing is available for the whiteboards in Teams meetings. The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

Author: Pushed commit with required changes.

`--CoreLoopSharingCapability [CoreLoopSharingCapability]`

> : Determines what level of sharing is available for the fluid on core partition. The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

Author: Pushed commit with required changes.

`--ContainerSharingCapability [ContainerSharingCapability]`

> : The valid values are: ExternalUserAndGuestSharing (default) - External user sharing (share by email) and guest link sharing are both enabled. Disabled - External user sharing (share by email) and guest link sharing are both disabled. ExternalUserSharingOnly - External user sharing (share by email) is enabled, but guest link sharing is disabled. ExistingExternalUserSharingOnly - Only guests already in your organization's directory. Allowed values `Disabled`, `ExternalUserSharingOnly`, `ExternalUserAndGuestSharing`, `ExistingExternalUserSharingOnly`.

Author: Pushed commit with required changes.

`--CoreDefaultShareLinkRole [CoreDefaultShareLinkRole]`
: Sets default share link role on core partition. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

`--CoreLoopDefaultSharingLinkRole [CoreLoopDefaultSharingLinkRole]`
: Sets default share link role for fluid on SharePoint sites. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

`--ContainerDefaultShareLinkRole [ContainerDefaultShareLinkRole]`

> : Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

Author: Pushed commit with required changes.

`--ContainerLoopDefaultShareLinkRole [ContainerLoopDefaultShareLinkRole]`

> : Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

Author: @milanholemans I am unable to find any information related to this option (`ContainerLoopDefaultShareLinkRole`). Can you please help?

`--OneDriveDefaultShareLinkRole [OneDriveDefaultShareLinkRole]`
: Sets default share link role on OneDrive partition. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

`--OneDriveLoopDefaultSharingLinkRole [OneDriveLoopDefaultSharingLinkRole]`
: Sets default share link role for fluid on OneDrive sites. Allowed values `Edit`, `View`, `LimitedEdit`, `LimitedView`, `ManageList`, `None`, `Owner`, `RestrictedView`, `Review`, `Submit`

`--CoreDefaultShareLinkScope [CoreDefaultShareLinkScope]`
: Sets default share link scope on core partition. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`

`--CoreLoopDefaultSharingLinkScope [CoreLoopDefaultSharingLinkScope]`
: Sets default share link scope for fluid on SharePoint sites. Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`

`--ContainerDefaultShareLinkScope [ContainerDefaultShareLinkScope]`

> : Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`

Author: Pushed commit with required changes.

`--ContainerLoopDefaultShareLinkScope [ContainerLoopDefaultShareLinkScope]`

> : Allowed values `Anyone`, `Organization`, `SpecificPeople`, `Uninitialized`

Author: @milanholemans I am unable to find any information related to this option (`ContainerLoopDefaultShareLinkScope`). Can you please help?

`--ReduceTempTokenLifetimeEnabled [ReduceTempTokenLifetimeEnabled]`
: PEnables reduced session timeout for temporary URLs used by apps for document download scenarios. Allowed values `true`, `false`

`--ReduceTempTokenLifetimeValue [ReduceTempTokenLifetimeValue]`
: Sets the session timeout value for temporary URLs. The value can be set between 5 and 15 minutes and the default value is 15 minutes

`--ShowOpenInDesktopOptionForSyncedFiles [ShowOpenInDesktopOptionForSyncedFiles]`
: Displays the "Open in desktop" option when users go to SharePoint or OneDrive on the web and open the shortcut menu for a file that they're syncing with the OneDrive sync app. The default value is false. Allowed values `true`, `false`

`--ShowPeoplePickerGroupSuggestionsForIB [ShowPeoplePickerGroupSuggestionsForIB]`
: Allows showing group suggestions for information barriers (IBs) in the People Picker. The default value is false. Allowed values `true`, `false`

`--SiteOwnerManageLegacyServicePrincipalEnabled [SiteOwnerManageLegacyServicePrincipalEnabled]`

> : Allows the site owner to manage the Azure Access Control (ACS) service principal. Allowed values `true`, `false`

Author: Pushed commit with required changes.

`--LimitedAccessFileType [LimitedAccessFileType]`
: Allows users to preview only Office files in the browser. This option increases security but may be a barrier to user productivity. Allowed values `OfficeOnlineFilesOnly`, `WebPreviewableFiles`, `OtherFiles`

`--MediaTranscription [MediaTranscription]`
: When enabled, videos can have transcripts generated on demand or generated automatically in certain scenarios. Allowed values `Enabled`, `Disabled`

`--MediaTranscriptionAutomaticFeatures [MediaTranscriptionAutomaticFeatures]`
: When enabled, videos can have transcripts generated automatically on upload. Allowed values `Enabled`, `Disabled`

`--ImageTaggingOption [ImageTaggingOption]`

> : Specifies the Syntex image tagging option for the tenant. Allowed values `Basic`, `Enhanced`, `Disabled`

Author: Pushed commit with required changes.

`--MarkNewFilesSensitiveByDefault [MarkNewFilesSensitiveByDefault]`
: If external sharing is turned on, sensitive content could be shared and accessed by guests before the Office DLP rule finishes processing, you can address this issue by configuring this option. Allowed values `AllowExternalSharing`, `BlockExternalSharing`

`--OCRAdminSiteListFileName [OCRAdminSiteListFileName]`

> : Sets name of the file which contains the list which contains OCR for full text search enabled sites by the tenant admin

Author: Pushed commit with required changes.

`--OCRComplianceSiteListFileName [OCRComplianceSiteListFileName]`

> : Sets name of the file which contains the list which contains OCR for full text search enabled sites by the compliance admin

Author: Pushed commit with required changes.

`--OCRModeForAdminSites [OCRModeForAdminSites]`

> : Specifies whether the tenant admin has enabled OCR on SharePoint sites for full text search through Syntex. Allowed values `Disabled`, `ExclusionList`, `InclusionList`

Author: Pushed commit with required changes.

`--OCRModeForComplianceODBs [OCRModeForComplianceODBs]`
: Specifies whether the compliance admin has enabled OCR for full text search through Syntex for ODBs. Allowed values `Disabled`, `ExclusionList`, `InclusionList`

`--OCRModeForComplianceSites [OCRModeForComplianceSites]`
: Specifies whether the compliance admin has enabled OCR for full text search through Syntex for SharePoint sites. Allowed values `Disabled`, `ExclusionList`, `InclusionList`

`--OneDriveDefaultLinkToExistingAccess [OneDriveDefaultLinkToExistingAccess]`
: Sets default share link to existing access on OneDrive partition. Allowed values `true`, `false`

`--ContainerDefaultLinkToExistingAccess [ContainerDefaultLinkToExistingAccess]`

> : Sets default share link to existing access on Containers. Allowed values `true`, `false`

Author: Pushed commit with required changes.

`--DisabledWebPartIds [DisabledWebPartIds]`
: Sets disabled web part Ids. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`

`--DisabledModernListTemplateIds [DisabledModernListTemplateIds]`

> : Sets disabled modern list templates Ids to hide when creating a new list. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`

Author: Pushed commit with required changes.

`--EnableAzureADB2BIntegration [EnableAzureADB2BIntegration]`
: Enables the preview for OneDrive and SharePoint integration with Azure AD B2B. Allowed values `true`, `false`. Azure AD one-time passcode needs to be enabled for this integration to work. For more information see [http://aka.ms/spo-b2b-integration](https://aka.ms/spo-b2b-integration).

`--EnableAutoNewsDigest [EnableAutoNewsDigest]`

> : Enables or disables SharePoint auto news digest. Allowed values `true`, `false`.

Author: Pushed commit with required changes.

`--AllowCommentsTextOnEmailEnabled [AllowCommentsTextOnEmailEnabled]`

> : Allows showing surrounding document context in email notification when user mentioned in comments. Allowed values `true`, `false`.

Author: Pushed commit with required changes.

`--CommentsOnFilesDisabled [CommentsOnFilesDisabled]`
: Disables or enables commenting functionality on files. Allowed values `true`, `false`.

`--DisableAddToOneDrive [DisableAddToOneDrive]`
: Disables or enables `Add shortcut to OneDrive` feature in SharePoint document libraries.  Allowed values `true`, `false`.

`--DisableBackToClassic [DisableBackToClassic]`

> : Disables or enables the back to classic link for libraries and lists. Allowed values `true`, `false`.

Author: Pushed commit with required changes.

`--DisablePersonalListCreation [DisablePersonalListCreation]`
: Allows creating personal lists within the OneDrive for Business site of the user. Allowed values `true`, `false`.

`--ViewInFileExplorerEnabled [ViewInFileExplorerEnabled]`
: Enables or disables the ability to use View in Explorer in Microsoft Edge. Allowed values `true`, `false`.

`--AllowGuestUserShareToUsersNotInSiteCollection [AllowGuestUserShareToUsersNotInSiteCollection]`
: Allows guest users to share to users not in the site collection. Allowed values `true`, `false`.

`--BlockSendLabelMismatchEmail [BlockSendLabelMismatchEmail]`
: Allows blocking of the automated e-mail being sent when somebody uploads a document to a site that's protected with a sensitivity label and their document has a higher priority sensitivity label than the sensitivity label applied to the site. Allowed values `true`, `false`.

`--CoreDefaultLinkToExistingAccess [CoreDefaultLinkToExistingAccess]`
: Sets default share link to existing access on core partition. Allowed values `true`, `false`.

`--CoreRequestFilesLinkEnabled [CoreRequestFilesLinkEnabled]`

> : Enables or disables the Request files link for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Enables or disables the Request files link on the core partition for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.
```

Author: Pushed commit with required changes.

: Enables or disables the Request files link on the OneDrive partition for all OneDrive sites. If this value is not set, Request files link will only show for OneDrives with Anyone links enabled. Allowed values `true`, `false`.

`--DisableDocumentLibraryDefaultLabeling [DisableDocumentLibraryDefaultLabeling]`
: Allows tenant admins to disable the capability of configuring a default sensitivity label for a document library. Allowed values `true`, `false`.

`--DisableVivaConnectionsAnalytics [DisableVivaConnectionsAnalytics]`
: Specifies whether Viva Connections Analytics is disabled or enabled on the tenant. Allowed values `true`, `false`.

`--DisplayNamesOfFileViewersInSpo [DisplayNamesOfFileViewersInSpo]`
: Allows configuring whether display name of people who view the file are visible in the property pane of the site in SharePoint site collection. Allowed values `true`, `false`.

`--EnableAIPIntegration [EnableAIPIntegration]`
:  Specifies whether Azure Information Protection (AIP) should be enabled on the tenant. Allowed values `true`, `false`.

`--EnableRestrictedAccessControl [EnableRestrictedAccessControl]`

> : Specifies whether the policy which determines Restricted Access Control Policy is enabled. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Enables restricted access control in SharePoint Online. You need to wait approximately 1 hour before managing restricted access control for a site collection. Allowed values `true`, `false`.
```

Author: Pushed commit with required changes.

> `--ExternalUserExpirationRequired [ExternalUserExpirationRequired]`

Reviewer: Can we place this option next to the option that configures how many days an expiration time is? This will make sure that related options are grouped together in the docs.

Author: Yes, will push the new commit for this suggestion.

> : Specifies whether to enable the external user expiration policy, where external users will be expired and removed from the site collection. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Specifies whether to enable the external user expiration policy, where external users will be expired and removed from the site collection in a given number of days. Allowed values `true`, `false`.
```

`--HideSyncButtonOnDocLib [HideSyncButtonOnDocLib]`

> : Enables or disables Sync button on all document libraries in SharePoint Team sites. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Enables or disables the "Sync" button on all document libraries in SharePoint sites and OneDrive for Business sites. Allowed values `true`, `false`.
```

`--IncludeAtAGlanceInShareEmails [IncludeAtAGlanceInShareEmails]`

> : Enables or disables At A Glance feature in the sharing e-mails. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Enables or disables At A Glance feature in the sharing e-mails. This provides the key points and time to read for the shared item if available. Allowed values `true`, `false`.
```

`--InformationBarriersSuspension [InformationBarriersSuspension]`
: Allows suspension of the information barriers feature in a Microsoft 365 tenant. Allowed values `true`, `false`.

`--IsFluidEnabled [IsFluidEnabled]`
: Specifies whether Fluid components are enabled or disabled in the tenant. Allowed values `true`, `false`.

`--IsWBFluidEnabled [IsWBFluidEnabled]`
: Specifies whether Whiteboard is enabled or disabled for OneDrive for Business users. Allowed values `true`, `false`.

`--IsCollabMeetingNotesFluidEnabled [IsCollabMeetingNotesFluidEnabled]`

> : Specifies whether CollabMeetingNotes Fluid Framework is enabled or disabled. Allowed values `true`, `false`.

Reviewer: ```suggestion
: Specifies whether Fluid components are enabled or disabled in Microsoft OneNote collaborative meeting notes. Allowed values `true`, `false`.
```
